### ISSUE TITLE
[Auth] Lambda trigger exception not forwarded when using social sign-in

### ISSUE THREAD

**Issue by dnys1** [aws-amplify/amplify-android#1649]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-android/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-android/discussions).

### Language and Async Model

Kotlin

### Amplify Categories

Authentication

### Gradle script dependencies

<details>

```groovy
implementation 'com.amplifyframework:aws-auth-cognito:1.28.3-rc'
```

</details>

### Environment information

<details>

N/A

</details>

### Describe the bug

https://github.com/aws-amplify/amplify-flutter/issues/1279

CX is not receiving exception message from Lambda trigger when using social sign-in. I was able to reproduce when using social sign-in only - regular sign-up and Hosted UI both returned the exception.

Is this expected behavior? And is there a way to expose these exceptions?

### Reproduction steps (if applicable)

1. Set up lambda trigger which throws an exception (I tested with pre-signup).
2. Set up social sign-in (I tested with Google).
3. Sign in with social login
4. Receive generic error message w/out Lambda exception.

### Code Snippet

```java
// Put your code below this line.

```

### Log output

<details>

```
// Put your logs below this line


```

</details>

### GraphQL Schema

<details>

```graphql
// Put your schema below this line


```

</details>

**Comment by Sunsiha** [comment]
any update on this?

**Comment by eeatonaws** [comment]
A fix for the issue has been released in Amplify Android 1.37.4.
